core/fifo.c

```diff
@@ -104,7 +104,12 @@ int uwsgi_master_fifo() {
 
 	char *path = uwsgi_fifo_by_slot();
 
-	unlink(path);
+	if (unlink(path)) {
+		if (errno != ENOENT) {
+			uwsgi_error("uwsgi_master_fifo()/unlink()");
+			// exit(1); // XXX: If unlink() failed, mkfifo() will probably as well
```

Comment:
Either exit() or don't, both are fine by me,  but let's not add commented out code please :)

Reply:
To be on the safe side better to don't exit though

Reply:
Sounds sensible - thanks. Also changed to `&& errno != ENOENT`.